packages/backend-modules/publikator/lib/cache/search.ts

```diff
@@ -58,11 +64,11 @@ const find = async (args: any, context: GraphqlContext) => {
 
   const fields = [
     'id',
-    'commit.strings.text',
+    // 'commit.strings.text',
```

Comment:
we removed this for now because it never worked right, correct?

Reply:
Oversight on my part. Jeremy and I tried to improve Publikator search